Documentation/ceph-dashboard.md

```diff
@@ -59,6 +59,26 @@ To get the dashboard to use hyperlinks that include your prefix, you can set the
       urlPrefix: /ceph-dashboard
 ```
 
+The port that the dashboard is served on may be changed from the defaults using
```

Comment:
I'd suggest combining the three dashboard settings into a single yaml example, then explain the settings in bullet points.